building/glass/aptbranch.py

```diff
@@ -4,7 +4,11 @@
 import re
 import subprocess
 
+import validate
+
 SETUP_DIR = "/h/setup-apt-branch/"
+CONFIG_PATH = "/h/apt-branch-config/branches.yaml"
+CONFIG_SCHEMA_NAME = "branches-schema.yaml"
```

Comment:
This is a great change to branches (at very first glance); please submit it first and separately.